### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by SteveLasker**
Cleaning up the readme, scenarios, manifest and referrers spec markdown files.

### CODE REVIEW

File: scenarios.md [hunk 1]
```diff
@@ -1,51 +1,36 @@
...
+## Reference Type Requirements
 
-### Migrating from Phase 1 to Phase 2
+Artifact reference types support the following requirements:
 
-Registries that implement phase 1 will likely focus on implementing reverse indexes, supporting the referrers api and lifecycle management to assure untagged reference types aren't automatically deleted.
-While the manifest format will change, the underlying capabilities for reverse indexes and lifecycle management will be maintained. 
+- Adding related artifacts, maintaining the original artifact digest and collection of associated tags.
+- `many:1` references, enabling multiple signatures, SBoMs, images with on-demand loading to be linked to a single artifact.
+- Build atop the existing distribution-spec push, persist and pull APIs.
```
Comment (michaelb990) [inline scenarios.md]:
```suggestion
- Build atop the existing distribution-spec push and pull APIs.
```

nit: Push & persist APIs are equivalent in my mind.

File: scenarios.md [hunk 2]
```diff
@@ -1,51 +1,36 @@
...
-It is not the intent to have dozens of manifest types.
-Phase 1 is a point in time release to meet pressing security requirements for supply chain artifacts, while building knowledge to feed into phase 2.
+## Reference Type Requirements
 
-### Migrating from Phase 1 to Phase 2
+Artifact reference types support the following requirements:
 
-Registries that implement phase 1 will likely focus on implementing reverse indexes, supporting the referrers api and lifecycle management to assure untagged reference types aren't automatically deleted.
-While the manifest format will change, the underlying capabilities for reverse indexes and lifecycle management will be maintained. 
+- Adding related artifacts, maintaining the original artifact digest and collection of associated tags.
```
Comment (michaelb990) [inline scenarios.md:15]: Not sure I understand what the second part of this means.

File: scenarios.md [hunk 3]
```diff
@@ -1,51 +1,36 @@
...
+- `many:1` references, enabling multiple signatures, SBoMs, images with on-demand loading to be linked to a single artifact.
+- Build atop the existing distribution-spec push, persist and pull APIs.
+- Avoiding the need to tag an artifact to assure its not garbage collected as an untagged manifest.
+- Discovery, based on the subject artifact. Consumers need not know the specific digest for a signature. Rather a registry client discovers a list of references based a provided tag or digest.
+- Copying a graph of references within and across ORAS Artifact enabled, distribution spec based registries, enabling an image, its signatures, SBoMs and images with on-demand loading to be copied as a collection.
```
Comment (michaelb990) [inline scenarios.md:20]: what does "on-demand loading" mean here? I'd suggest removing it and focusing on the ability to copy an image & its related artifacts between registries.

File: scenarios.md [hunk 4]
```diff
@@ -197,18 +171,23 @@ The  `net-monitor:v1` SBoM will also be signed, providing yet another leaf node.
...
-The `net-monitor:v1` image is represented as an oci-image, and requires no changes to its manifest to support the enhancements.
+The Notary v2 signature and SBoM reference the `net-monitor:v1` container image (as a digest) through the `subject` property.
+The `net-monitor:v1` container image is represented as an oci-image, and requires no changes to its manifest to support the enhancements.
 The directionality of the `subject` reference enables links to existing content, without changing the existing content.
 
 ### Deletion and Ref Counting
 
 The `subject` reference is a hard reference.
-Just as the layers of an OCI Image are deleted (*ref-counted -1*), any artifacts with a `subject` referring to the target manifest SHOULD be deleted (*ref-counted -1*).
+Just as the layers of an OCI Image are deleted (*ref-counted -1*), any artifacts with a `subject` referring to the target manifest MAY be deleted (*ref-counted -1*).
```
Comment (michaelb990) [inline scenarios.md]: In other parts of the spec, we've removed the words that imply a certain GC algorith like "ref-count". Should we do that here?

File: scenarios.md [hunk 5]
```diff
@@ -221,36 +200,58 @@ The main scenarios include:
 
 ### Content Discovery
 
-Registries support a list of content within a repository.
-A container image, multi-arch container image, Helm Chart, WASM and other ORAS Artifact types can be listed based on their `manifest.config.mediaType`
+Registries support listing content within a repository.
+A container image, multi-arch container image, Helm Chart, WASM and other ORAS Artifact types may be listed based on their `manifest.config.mediaType`.
+
+### Flat Listings
```
Comment (michaelb990) [inline scenarios.md:206]: Why are we adding this? I'm not sure it belongs in the spec.

File: scenarios.md [hunk 6]
```diff
@@ -221,36 +200,58 @@ The main scenarios include:
...
 
-In the above example, all the artifacts are displayed without relation to each other.
-The image signature, SBoM, SBoM signature, Helm signature are listed with digests as they have no individual identity. However, the registry has no knowledge these artifacts are references of the image, SBoM or Helm chart.
+### Hierarchical Listings
+
+In the following example, the Notary v2 signature, an SBoM and a collection of attributes may be displayed as hierarchical graph.
+
+The references may be expanded or collapsed as the `oras.artifact.manifest` provides reference information, based on the `artifact.manifest.subject` property.
 
 ![flat listing of ORAS Artifacts](media/repo-listing-attributed.svg)
```
Comment (michaelb990) [inline scenarios.md]: alt-text should be changed here i think

**Review by michaelb990 — CHANGES_REQUESTED** [review]
It was hard to follow some of this because there are formatting changes mixed with changes in the language. Left a few comments, tried to focus on the language changes.

File: scenarios.md (hunk 2 above)
Comment (SteveLasker) [reply scenarios.md:15]: One of the goals of the Artifacts Spec is the ability to add links to existing content. That's important as it doesn't break existing workflows that are based on the digest and/or tag of the original content.
If you push `net-monitor:v1` with a digest of `sha256:abc123`, and you have a helm chart that references it, you need that to be maintained.
If you add an SBoM or a signature, the helm chart shouldn't change it's digest or tag reference. 

Put that in a bullet :)

File: scenarios.md (hunk 3 above)
Comment (SteveLasker) [reply scenarios.md:20]: it's a nod to the nydus folks that are also leveraging the Aritfact Manifest/Reference Types to support their scenario. 
See: [net-monitor-image-nydus-ondemand-loading.json](https://github.com/oras-project/artifacts-spec/blob/main/examples/net-monitor-image-nydus-ondemand-loading.json) for an example.

It looks like others that are also doing expansion will use this, like Alibabas DADI implementation.

File: scenarios.md (hunk 5 above)
Comment (SteveLasker) [reply scenarios.md:206]: Which part are you asking about? The flat listing?

**Review by michaelb990 — APPROVED** [review]

**Comment by michaelb990** [comment]
Make sure to squash-merge or clean up commits before merging.

**Review by justincormack — APPROVED** [review]

**Review by sajayantony — APPROVED** [review]
